app/templates/events/eventList.html

```diff
@@ -147,25 +153,25 @@ <h1 class="text-center">Events List for {{selectedTerm.description}}</h1>
       {% endfor %}
       {% if selectedTerm == g.current_term and events[-1].isPastStart %}
         <tr>
-          {% set colspan_value = 5 if type == 'bonner' or type == 'studentLed' else 6 %}
+          {% set colspan_value = 5 if type == 'bonner' or type == 'volunteerOpportunities' else 6 %}
           <td colspan="{{colspan_value}}" class="p-3">There are no upcoming events for this program</td>
         </tr>
       {% endif %}
      </tbody>
    </table>
  </div>
-{% elif type != "studentLed" %}
+{% elif type != "volunteerOpportunities" %}
   <p>There are no events for this program</p>
 {% endif %}
 {% endmacro %}
 
-<!-- #############  Student Led Services  ############# -->
+<!-- #############  Volunteer Opportunities   ############# -->
 
 <div class="tab-content" id="pills-tabContent">
-  <div class="tab-pane fade show {{'active' if activeTab == 'studentLedEvents' else ''}}" id="pills-student-led" role="tabpanel" aria-labelledby="pills-student-led-tab">
-    {% if studentLedEvents %}
+  <div class="tab-pane fade show {{'active' if activeTab == 'volunteerOpportunities' else ''}}" id="pills-student-led" role="tabpanel" aria-labelledby="pills-student-led-tab">
```

Comment:
Change id and aria-labelledby from "student-led" language and chnage all the instances where such id is mentioned